### PR TITLE
add optional resultsPerPage parameter to styledOutputService

### PR DESCRIPTION
**Pull request by MikeVanMourik**
add optional resultsPerPage parameter to styledOutputService
Asana: none, just good to have

### CODE REVIEW

File: Api/Modules/Queries/Services/StyledOutputService.cs [hunk 1]
```diff
@@ -145,9 +148,12 @@ public async Task<ServiceResult<JToken>> GetStyledOutputResultJsonAsync(ClaimsId
             
             clientDatabaseConnection.ClearParameters();
 
+            int pageResultCount = resultsPerPage > 0 ? resultsPerPage : resultsDefaultPerPage;
+            pageResultCount = pageResultCount > maxResultsPerPage ? maxResultsPerPage : pageResultCount;
```
Comment (MarkVanDijkHG) [inline Api/Modules/Queries/Services/StyledOutputService.cs:152]: Isn't it more readable to use the Math.Min function here?

**Review by MarkVanDijkHG — CHANGES_REQUESTED** [review]
In multiple methods the parameter "resultsPerPage" is set to have a default of 0 if not filled. Later on in the code it is set to a default value if the provided value was 0. Isn't it more logical to set the default value in the methods to the default value used later on? Then it is directly clear what the default is and won't cause confusion because the endpoint makes it look like that if no value is provided you won't get any result.

File: Api/Modules/Queries/Services/StyledOutputService.cs (hunk 1 above)
Comment (MikeVanMourik) [reply Api/Modules/Queries/Services/StyledOutputService.cs:152]: im just in love with the short hand if statement, its been swapped out :)

**Comment by MikeVanMourik** [comment]
tried to keep all the settings in the same class, but that makes no sense ofcourse since the controller is the part that is exposed to the user anyway, i changed things around to make the pagecount no longer optional in the classes and filled in the default value in the controller

**Comment by MikeVanMourik** [comment]
also noticed 2 files were included that were not in use, removed those as well

**Review by MarkVanDijkHG — APPROVED** [review]

**Pull request merged**